README.md

```diff
@@ -25,6 +25,12 @@ Read more about the project in the latest article about Bootleg on [Medium](http
 
 <br/>
 
+## Project overview

```

Comment:
I've been roughly following the pattern set by sojourn https://github.com/ConsenSys/web3studio-sojourn/issues/42 for all of our readmes. 

Vaguely I would think this content belongs with the stuff on line 19 but I can see it as it's own section. 

When adding new sections, make sure to update the table of contents at the top (around line 14)